.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Auto Author Assign'
+name: Auto Author Assign
```

Comment:
These yaml changes are cosmetic. I don't want to take the cosmetic changes.

Could you revert these changes?

Reply:
@toshimaru 
ok I rollback yaml files.

but, I recommend to read this~
https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml

yaml syntax no quotation; because string is default.

Reply:
Thanks for the information. :)

Okay, I'm gonna take this change in another PR. 👍

Reply:
I've took your suggestion in #81 (co-authored by you).

Thanks for your suggestion 👍